### PR TITLE
Update Bundler version to 1.16.2 in Travis config file

### PR DESCRIPTION
**Pull request by ghost**
# (test)

### CODE REVIEW

**Comment by lucascaton** [comment]
some comment (test)

**Comment by ghost** [comment]
test

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 language: ruby
 
-before_install: gem install bundler -v 1.15.4
+before_install: gem install bundler -v 1.16.2
```
Comment (lucascaton) [inline .travis.yml:3]: test!

**Review by lucascaton — CHANGES_REQUESTED** [review]
please change something (test)

**Review by lucascaton — APPROVED** [review]
👍

**Pull request merged**